src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw {
+    private final String color;
```

Comment:
why final? it may be like that in our case, but overall better to not do it without the explicit requirements